build.log

```diff
@@ -0,0 +1,60 @@
+[4B[0K
+[5A[1m<[0;1m-------------> 1% EXECUTING [44s][m[34D[1B> IDLE[6D[1B> IDLE[6D[1B> IDLE[6D[1B> IDLE[6D[1B[5A[0K
+[31mFAILURE: [39m[31mBuild failed with an exception.[39m[0K
+[0K
+* What went wrong:[0K
+Execution failed for task ':proto:generateProto'.[0K
+[33m> [39mjava.io.IOException: Cannot run program "/build/tmp.YollhrLx8O/caches/modules-2/files-2.1/com.google.protobuf/protoc/3.19.1/d231c807be82ffc29b02525c8c2691772eb945bc/protoc-3.19.1-linux-x86_64.exe": error=2, No such file or directory
```

Comment:
HELP_WANTED: Spent 7 hours on this and I have no idea how to fix this.. It seems to only happen on NixOS

RFC @obfusk @linsui You seem to have been dealing with problems alike while packaging apksigner, your input to this would be very appreciated

Reply:
Do we? You should first check if the file is there. If it's there, then the problem is that it tries to run a binary on NixOS. You should patch something so that the protoc in nixpkgs is used instead.

Reply:
https://github.com/google/protobuf-gradle-plugin?tab=readme-ov-file#locate-external-executables I thought you just need to add protoc to nativeBuildInputs.